plugins/arDominionB5Plugin/scss/_typography.scss

```diff
@@ -0,0 +1,32 @@
+@import "@fontsource/open-sans/scss/mixins";
+
+// Add font-face declarations for all Open Sans weights and styles
+@include faces($subsets: all, $weights: all, $styles: all);
+
+:root {
+  --bs-font-sans-serif: "Open Sans", sans-serif;
+}
+
+b,
+strong {
+  font-weight: 700;
+}
```

Comment:
Can this be changed with these vars? https://github.com/artefactual/atom/blob/qa/2.x/plugins/arDominionB5Plugin/scss/_variables.scss#L44-L47

In fact, removing those vars may do what you are doing directly.